src/test/java/co/pragra/attendancesystem/controller/CohortControllerTest.java

```diff
@@ -0,0 +1,91 @@
+package co.pragra.attendancesystem.controller;
```

Comment:
Pls. refactor to be renamed CohortApiTest.java and move to co.pragra.attendancesystem.rest package in src/test/java